examples/eks_test_fixture/main.tf

```diff
@@ -67,14 +67,13 @@ module "vpc" {
 }
 
 module "eks" {
-  source             = "../.."
-  cluster_name       = "${local.cluster_name}"
-  subnets            = ["${module.vpc.public_subnets}", "${module.vpc.private_subnets}"]
-  tags               = "${local.tags}"
-  vpc_id             = "${module.vpc.vpc_id}"
-  worker_groups      = "${local.worker_groups}"
-  worker_group_count = "1"
-  map_roles          = "${var.map_roles}"
-  map_users          = "${var.map_users}"
-  map_accounts       = "${var.map_accounts}"
+  source        = "../.."
+  cluster_name  = "${local.cluster_name}"
+  subnets       = ["${module.vpc.public_subnets}", "${module.vpc.private_subnets}"]
+  tags          = "${local.tags}"
+  vpc_id        = "${module.vpc.vpc_id}"
+  worker_groups = "${local.worker_groups}"
+  map_roles     = "${var.map_roles}"
+  map_users     = "${var.map_users}"
+  map_accounts  = "${var.map_accounts}"
```

Comment:
#75

Reply:
This is just `terraform fmt` changes. Nothing related to that PR?

Reply:
#75 originally had `worker_group_count = "1"` set here. Technically unnecessary as that's the default value.

Reply:
Ahh I see what you mean. Correct. I think it should stay then.